pkg/autoscaler/autoscaler.go

```diff
@@ -249,11 +250,26 @@ var _ = Describe("[Feature:Machines] Autoscaler should", func() {
 		// node.
 		workloadMemRequest = resource.MustParse(fmt.Sprintf("%v", 0.7*float32(bytes)))
 
+		By("Starting Cluster Autoscaler event watcher")
```

Comment:
Why does this commit needs to go in this PR?

Reply:
Can we have a different PR discussion for that? I'd be in favour of not relying on core autoscaler events for e2e testing which might break e2e on each rebase.

Reply:
sure, i can break that out into a different PR.

my reasoning for adding it in at the top level is that i've found it invaluable in debugging failures. it is not used for any testing purposes outside of the "scale up and down" test, it will print the autoscaler events as they are happening during the test.

Reply:
created #163